src/flint/flintlib/functions/fmpz_vec.pxd

```diff
@@ -68,3 +68,6 @@ cdef extern from "flint/fmpz_vec.h":
     void _fmpz_vec_dot_general_naive(fmpz_t res, const fmpz_t initial, int subtract, const fmpz_struct * a, const fmpz_struct * b, int reverse, slong len)
     void _fmpz_vec_dot_general(fmpz_t res, const fmpz_t initial, int subtract, const fmpz_struct * a, const fmpz_struct * b, int reverse, slong len)
     void _fmpz_vec_dot(fmpz_t res, const fmpz_struct * vec1, const fmpz_struct * vec2, slong len2)
+
+    # Undocumented internal functions
+    void _fmpz_vec_content(fmpz_t res, const fmpz_struct * vec, slong len)
```

Comment:
We can't add these in the `functions/*.pxd` because these files are now all auto-generated by:
```console
$ bin/all_rst_to_pxd.sh ../flint/doc/source/
```
It will need to go in `flintlib/types` somewhere or otherwise would need to be special-cased in the `rst_to_pxd` script.

Reply:
See:
https://github.com/flintlib/python-flint/blob/14241cdd5c580b74eab2b9e010f8c0a0ae7e222c/src/flint/flintlib/README.md?plain=1#L1-L35

Reply:
Makes sense. Looks like this function was picked up by the recent auto-generation so I've just removed it instead.